### ISSUE TITLE
Offer Language Server, HABPanel as add-ons

### ISSUE THREAD

**Issue by dilyanpalauzov** [openhab/openhab-core#3041]
I run openHab 3.2 on Raspberry Pi 3.  I do not use the habpanel, language, voice.  Please expose the features: voice control/voice rest interface, HABPanel UI, language server, Basic UI  as installable add-ons.

This way openHAB will load faster and use less RAM.

**Comment by dilyanpalauzov** [comment]
I mean as addons, which can be installed from the Market Place.

**Comment by dilyanpalauzov** [comment]
Or addons, which are initially installed, but the user can remove them over Main UI/Market Place. Ideally, after upgrade, these addons shall not migacally reinstall theirselves.

**Comment by dilyanpalauzov** [comment]
In the Main UI →Settings → UIs I see options to change the theme to dark/standard, enable/disable icons, compact view, capitalize values, default sitemap.  I see no option to about HABPanel.  I have HABPanel installed, since it shown on the main URL (`:8080/`) under “Other applications”, together with Basic UI.  In any case, from the Market Place →“Other UI Add-ons” I could deinstall it, and BasicUI can also be removed from there.

Concerning the voice controls and language server (openHAB Core :: Bundles :: Voice REST Interface; openHAB Core :: Bundles :: Language Server; openHAB Core :: Bundles :: Voice), they can be disabled from the Karaf console, but as you said things might break. Having them preinstalled is useful for new users, I agree.  Nevertheless, I think it is good idea, if these are managed as add-ons, which can be removed/re-added from the Market Place.  This way shall be assured, that removing them will not crash the system (discontinued code highlighting is not a crash).

To sum up, I propose that some bundles from core are offered as add-ons, which addons are installed by default, but users can uninstall them over the Market Place.  This way users can get a system, loading faster and needing less RAM.  Good candidates for bundles, which shall be converted to add-ons, are Language Server, Voice, Voice REST Interface.

**Comment by J-N-K** [comment]
Since nobody objected against closing this issue, I'll tag it as "won't fix". Anyway, thank you for your suggestion. We'll keep in mind that small systems might benefit from adding new features as optional bundles in the future.
